opsmngr/agent_api_keys.go

```diff
@@ -0,0 +1,91 @@
+package opsmngr
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+
+	atlas "github.com/mongodb/go-client-mongodb-atlas/mongodbatlas"
+)
+
+const (
+	agentAPIKeysBasePath = "groups/%s/agentapikeys"
+)
+
+type AgentAPIKeysService interface {
+	Create(context.Context, string, AgentAPIKeysRequest) (*AgentAPIKey, *atlas.Response, error)
+	List(context.Context, string) (*[]AgentAPIKey, *atlas.Response, error)
+	Delete(context.Context, string, string) (*atlas.Response, error)
+}
+
+type AgentAPIKeysServiceOp struct {
+	Client atlas.RequestDoer
+}
+
+type AgentAPIKey struct {
+	ID            string  `json:"_id"`
+	Key           string  `json:"key"`
+	Desc          string  `json:"desc"`
+	CreatedTime   int64   `json:"createdTime"`
```

Comment:
The documentation says that is a string but is an int instead

Reply:
int? timestamp epoch kind of ting?